packages/lesswrong/components/posts/TableOfContents/TableOfContentsDivider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { registerComponent } from "../../../lib/vulcan-lib";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  divider: {
+    width: 80,
+    marginBottom:theme.spacing.unit,
+    marginRight: "auto",
+    borderBottom: theme.palette.border.faint,
+    paddingBottom: theme.spacing.unit,
+    display:"block",
+  }
+})
+
+const TableOfContentsDivider = ({classes}: {
+  classes: ClassesType,
+}) => {
+  return <div/>
```

Comment:
This doesn't seem to be using the styles specified above?

Reply:
Whoops, looks like I may have dropped a className in e6ac31bf27cee4af905ac5f0512c7cdd0c2a4e4a. Fixed.